doc/internal/man3/OPTIONS.pod

```diff
@@ -214,34 +227,40 @@ end of arguments has been reached. This is typically done like this:
         }
     }
 
-The opt_help() function takes a list of option definitions and prints a
-nicely-formatted output.
+Within the option parsing loop, the following functions may be called.
+
+The opt_flag() fuction returns the most recent option name
```

Comment:
Typo function

Reply:
It turns out that this file had three such typos :) Apparently by copy&paste. Fixed.